index.js

```diff
@@ -0,0 +1,241 @@
+const express = require("express")
+const helmet = require("helmet")
+const knex = require("knex")
+const knexConfig = require("./knexfile.js")
+const cors = require("cors")
+const db = knex(knexConfig.development)
+const projectdb = require("./data/helpers/projectsDB")
+
+const server = express()
+
+server.use(express.json())
+server.use(cors())
+server.use(helmet())
+
+function logger(req, res, next) {
+  console.log(`${req.method} to ${req.url}`)
+
+  next()
+}
+
+server.use(logger)
+
+server.get("/", (req, res) => {
+  res.send("Working")
+})
+
+// get projects
+server.get("/api/projects", (req, res) => {
+  db("projects")
+    .then(projects => {
+      res.status(200).json(projects)
+    })
+    .catch(err => {
+      console.log(err)
+      res.status(500).json({ error: `Something went wrong.` })
+    })
+})
+
+// get project ID
+server.get("/api/projects/:id", (req, res) => {
+  const { id } = req.params
+
+  db("projects")
+    .where("id", id)
+    .then(project => {
+      if (!project.length) {
+        res.status(404).json({ error: `Could not find project with ID ${id}` })
+      }
+      db("actions")
+        .where("project_id", id)
+        .then(actions => {
+          project[0].actions = actions
+
+          res.status(200).json(project)
+        })
+        .catch(err => {
+          console.log(err)
+          res
+            .status(500)
+            .json({ error: `An error occured retrieving project ${id}.` })
+        })
```

Comment:
This `catch` probably ought to be on the call to the `projects` table rather than the one to the `actions` table.